scss/civicrm/_variables.scss

```diff
@@ -41,3 +48,5 @@ $fa-var-info-msg: '\f05a';
 $fa-var-success: '\f00c';
 $fa-caret-down: '\f078';
 $fa-caret-right: '\f054';
+$fa-chevron-down: '\f078';
```

Comment:
We have this variables defined somewhere use that directly instead of defining here